docs/src/contributing.md

```diff
@@ -0,0 +1,112 @@
+# Contributing
+
+## Tool Setup
+
+### `python`
+
+To contribute code or documentation updates, an installation of Python 3 is required.
+
+### `hatch`
+
+This project utilizes [`hatch`](https://hatch.pypa.io/latest/) to manage Python environments for development and testing.  Follow
+[the `hatch` installation instructions](https://hatch.pypa.io/latest/install/) before continuing through this document.
+
+### `pre-commit`
+
+Additionally, this project uses [`pre-commit`](https://pre-commit.com/) Git hooks to run linting and formatting checks against each commit.  See [the `pre-commit` installation instructions](https://pre-commit.com/#install) for how to install this tool.
+
+Once installed, run `pre-commit install` to set up the git hook scripts:
```

Comment:
This comment is another note to myself - I don't believe I've configured pre-commit on this repo post-fork. Will need to do that soon.

Reply:
Done. Precommit has been configured on this repo.